gui/autobutcher.lua

```diff
@@ -316,7 +323,7 @@ function WatchList:init()
             },
             widgets.Panel{
                 view_id='footer',
-                frame={l=0, r=0, b=0, h=6},
+                frame={l=0, r=0, b=0, h=7},
```

Comment:
if you change `h` here, you'll also need to change the `b` value for `list_panel`